python/lsst/meas/modelfit/truncatedGaussian.cc

```diff
@@ -0,0 +1,137 @@
+// -*- lsst-c++ -*-
+/*
+ * LSST Data Management System
+ * Copyright 2008-2013 LSST Corporation.
+ *
+ * This product includes software developed by the
+ * LSST Project (http://www.lsst.org/).
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the LSST License Statement and
+ * the GNU General Public License along with this program.  If not,
+ * see <http://www.lsstcorp.org/LegalNotices/>.
+ */
+
+#include "pybind11/pybind11.h"
+#include "pybind11/stl.h"
+
+#include "numpy/arrayobject.h"
+#include "ndarray/pybind11.h"
+
+#include "lsst/meas/modelfit/TruncatedGaussian.h"
+
+namespace py = pybind11;
+using namespace pybind11::literals;
+
+namespace lsst {
+namespace meas {
+namespace modelfit {
+namespace {
+
+using Sampler = TruncatedGaussianSampler;
+using Evaluator = TruncatedGaussianEvaluator;
+using LogEvaluator = TruncatedGaussianLogEvaluator;
+
+using PyTruncatedGaussian = py::class_<TruncatedGaussian, std::shared_ptr<TruncatedGaussian>>;
+using PySampler = py::class_<Sampler, std::shared_ptr<Sampler>>;
+using PyEvaluator = py::class_<Evaluator, std::shared_ptr<Evaluator>>;
+using PyLogEvaluator = py::class_<LogEvaluator, std::shared_ptr<LogEvaluator>>;
+
+template <typename Class, typename PyClass>
+PyClass declareEvaluator(py::module & mod, std::string const & name) {
```

Comment:
Comments added. The other overload of `operator()` was not wrapped because it's just an Eigen version of the ndarray one, and hence they look the same in Python.